config/processCcdWithFakesDriver.py

```diff
@@ -1,9 +1,12 @@
 """
-Subaru-specific overrides for ProcessCcdWithFakesDriverTask
-(applied before HSC-specific overrides).
+HSC-specific overrides for ProcessCcdWithFakesDriverTask
 """
 import os.path
 
-
 config.processCcdWithFakes.load(os.path.join(os.path.dirname(__file__), "processCcdWithFakes.py"))
 config.ccdKey = 'ccd'
+
+config.doMakeSourceTable = True
+config.doSaveWideSourceTable = True
+config.transformSourceTable.load(os.path.join(os.path.dirname(__file__), "transformSourceTable.py"))
+config.ignoreCcdList = [9, 104, 105, 106, 107, 108, 109, 110, 111]
```

Comment:
Again, not for this ticket, but it seems that, except for the load, these are the same settings as in `processCcdDriver.py` and, if they are meant to be the same, it would probably be better if they could be inherited from there to avoid things getting out of sync (@sr525?).